bool can_release
bool assembly_all_done
bool cam_done
bool part1
bool part2
bool part3
